### PR TITLE
File structure

### PR DESCRIPTION
**Pull request by knanahassouna1**
# File Structure 
file structure contain express server and react js server to serve react js app .
## how to use it :
- how to run server visit #10 (initial server issue)
- to check the files arch visit #19 (file structure issue)

### CODE REVIEW

File: src/server/controllers/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const Router = require('express').Router();
```
Comment (mohannadhanafi) [inline src/server/controllers/index.js]: Good job :+1: 
but I think it's better to make another const
like this :
```
const express = require ('express');
const router = express.Router();
```

File: src/client/src/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
```
Comment (ishak52) [inline src/client/src/index.js]: App file doesn't exist

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+node_modules
```
[the remark this answers is not in the document]
Comment (knanahassouna1) [reply .gitignore]: i will remove it

File: src/client/public/index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="theme-color" content="#000000">
+    <!--
+      manifest.json provides metadata used when your web app is added to the
+      homescreen on Android. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
```
[the remark this answers is not in the document]
Comment (knanahassouna1) [reply src/client/public/index.html]: i will remove it

File: src/client/public/index.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
[the remark this answers is not in the document]
Comment (knanahassouna1) [reply src/client/public/index.html]: i will do it

File: src/client/src/index.js (hunk 2 above)
Comment (knanahassouna1) [reply src/client/src/index.js]: i will do it

**Pull request merged**